akka-discovery/src/test/resources/reference.conf

```diff
@@ -0,0 +1 @@
+akka.actor.serialize-messages=on
```

Comment:
in those it is important to have the trailing line breaks (probably doesn't matter here, but better to not risk trouble)